NitroxServer/Serialization/World/PersistedWorldData.cs

```diff
@@ -34,7 +34,6 @@ public static PersistedWorldData From(World world)
                 WorldData =
                 {
                     ParsedBatchCells = world.BatchEntitySpawner.SerializableParsedBatches,
-                    ServerStartTime = world.TimeKeeper.ServerStartTime,
```

Comment:
Ok yeah we can delete `TimeKeeper.cs` but we probably still need the ServerStartTime saved somewhere xD